vnext/PropertySheets/React.Cpp.props

```diff
@@ -27,9 +27,9 @@
     <HERMES_Version Condition="'$(HERMES_Version)' == ''">0.1.6</HERMES_Version>
     <HERMES_Package Condition="'$(HERMES_Package)' == ''">$(SolutionDir)packages\ReactNative.Hermes.Windows.$(HERMES_Version)</HERMES_Package>
 
-    <USE_V8 Condition="'$(USE_V8)' == ''">false</USE_V8>
-    <V8_Version Condition="'$(V8_Version)' == ''">0.1.6</V8_Version>
-    <V8_Package Condition="'$(V8_Package)' == ''">$(SolutionDir)packages\ReactNative.V8JSI.Windows.$(V8_Version)</V8_Package>
+    <USE_V8 Condition="'$(USE_V8)' == '' AND '$(Platform)' != 'ARM' AND '$(Platform)' != 'ARM64'">true</USE_V8>
```

Comment:
>true [](start = 98, length = 4)

Do we enable the V8 default? The description says that it is not fully tested yet and defaults are not changed.

Reply:
We enable build support for V8 (the library will link against the delay-loaded v8jsi.dll), but the default choice at runtime doesn't change.

---
In reply to: [387943114](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387943114) [](ancestors = 387943114)